h2o-algos/src/test/java/hex/ensemble/StackedEnsembleTest.java

```diff
@@ -879,4 +885,56 @@ private void test_SE_scoring_with_missing_response_column(Algorithm algo) {
         }
 
     }
+
+  @Test
+  public void testMissingValidationColumn() {
+    GBMModel gbmModel = null;
+    try {
+      Scope.enter();
+
+      final Frame trainingFrame = TestUtil.parse_test_file("./smalldata/iris/iris_wheader.csv");
+      Scope.track(trainingFrame);
+      final Frame partialFrame = TestUtil.parse_test_file("./smalldata/iris/iris_wheader.csv", new int[]{4}); // Missing fold column
+      Scope.track(partialFrame);
+
+      GBMModel.GBMParameters parameters = new GBMModel.GBMParameters();
+      parameters._train = trainingFrame._key;
+      parameters._fold_column = "class";
+      parameters._seed = 0xFEED;
+      parameters._response_column = "petal_len";
+      parameters._ntrees = 1;
+      parameters._keep_cross_validation_predictions = true;
+
+      GBM gbm = new GBM(parameters);
+      gbmModel = gbm.trainModel().get();
+      assertNotNull(gbmModel);
+      
+
+      final StackedEnsembleParameters seParams = new StackedEnsembleParameters();
+      seParams._train = partialFrame._key;
+      seParams._response_column = "petal_len";
+      seParams._metalearner_algorithm = Algorithm.AUTO;
+      seParams._base_models = new Key[]{gbmModel._key};
+      seParams._seed = 0xFEED;
+      seParams._metalearner_fold_column = "class";
```

Comment:
@Pscheidl so to reproduce this use-case, the user needs to explicitly specify a `_metalearner_fold_column` that doesn't exist in the `train` pr `blending` dataset? 
Then, I'd rather suggest to validate it in the same place as we check that it matches base models `fold_column`, that is in `StackedEnsemble.checkAndInheritModelProperties()`: having parameters validation spread around the code base gets quickly confusing.

Reply:
Although, changing my mind, may be better in `prepareLevelOneFrame` you're right, as it should be checked for `validation` and `blending` frames as well. Yeah, our parameters validation sucks.

Reply:
Moved to init phase. WDYT ?